### ISSUE TITLE
Python Bindings?

### ISSUE THREAD

**Issue by xrkk** [SVF-tools/SVF#407]
Do you have any plans to create any Python Bindings so that can be easily intergated into any tool-chain?

**Comment by yuleisui** [comment]
Yes, we plan to have the analysis results in fat bcs, so that the users can use existing python bindings to read the results. https://github.com/numba/llvmlite

You are also encouraged to help with this process.

**Comment by mbarbar** [comment]
There are now Python bindings available [here](https://github.com/SVF-tools/SVF-Python).
